### ISSUE TITLE
reddit

### ISSUE THREAD

**Issue by rockfordlhotka** [MarimerLLC/cslaforum#900]
I've set up a CSLA community on reddit as an experiement:

https://www.reddit.com/r/cslanet/

I've generally avoided reddit forever, because it was populated by much of the seedier side of the Internet. And maybe it still is, but I've found quite a number of communities hosted there that I rely on (for things like Elite Dangerous) and apparently the whole thing isn't toxic.

Of course there's a [csla tag on StackOverflow](https://stackoverflow.com/search?q=user:438576+[csla]) too, and that community is also pretty toxic in many ways.

So I guess just because part of a site or community is toxic doesn't mean it must be totally shunned.

The thing that makes SO pretty useless for forum-like scenarios is that their moderators don't allow anything that can't be answered with a code snippet. No questions about best practices or anything that could result in opinion-based answers.

Which is fine - that model is working for all of us - as long as we have somewhere else for broader dialog.

We've been using this GitHub repo as a forum for quite a long time now. I'm not sure it is the worst tool, but it isn't the best tool either. I kind of miss a lot of what we had in the previous forum software - but I'm also not about to self-host a forum again!!

In short, I thought I'd set up a reddit community and see what happens. We'll see 😺

**Comment by Chicagoan2016** [comment]
@rockfordlhotka, we could experiment on reddit but  some of the reddit communities/moderators could  be pretty rude : ).
I like our current forum here, our fellow members have been so helpful to me, I can't thank them enough.

**Comment by rockfordlhotka** [comment]
Thanks @Chicagoan2016 - that's the sort of feedback I'm looking for.

**Comment by Chicagoan2016** [comment]
@rockfordlhotka , I want to share something about my recent experience with stackoverflow. I have mentioned 'Prism' here about a week or so ago, I had looked at it in the past and I liked it. I wanted to share my experience with my team, as no one in my team has looked at Prism I wanted to show my team some resources to learn Prism. I went to the Prism github repo/forum and they forwarded me to Stackoverflow for questions, I asked a simple question "Where should our team start learning Prism?".
Since it was a very simple question and didn't align with the 'genius' of Stackoverflow, it was closed.

Kind regards

**Comment by Chicagoan2016** [comment]
Oh My, here is the funny part, Dan Siegel who is one of the Prism maintainers commented on my question, agreeing it's a valid question :) :)

**Comment by rockfordlhotka** [comment]
Yeah, SO is _horrible_ for general forum scenarios - they don't want to be a forum, just a Q&A site for discrete questions.

**Comment by rockfordlhotka** [comment]
I just miss the ... feeling of community that we had in the previous forum software.

This GH "forum" is functional - provides the minimum features necessary - but it doesn't seem to have engendered the same sense of community that existed in the past.

Now that might have nothing to do with the software at all - times have changed, I spend years recovering from illness - so there are a lot of factors that probably go into all this.

**Comment by Chicagoan2016** [comment]
@rockfordlhotka , as your student, I would like to know how are you feeling now?
Not trying to be intrusive, wishing you great health.

Kind regards

**Comment by rockfordlhotka** [comment]
Thank you for asking. I'm doing very well. I'm in a high risk group with this whole C19 thing, so I'm being careful, but beyond that I'm able to do the majority of the things I enjoyed before all the surgeries.
